src/routes/committee/suggestions/+page.svelte

```diff
@@ -87,7 +88,7 @@
 			{#if song.liked || !filterFavourites}
 				<StructuredListRow>
 					<StructuredListCell>
-						{data.users.find((user) => user.id === song.user.id)?.name ?? 'Unknown'}
+						{(data.users.find((user) => user.id === song.user.id) ?? unknownUser).name}
```

Comment:
tha hell, ik geloof niet dat dit niet kan blijven zoals het was

Reply:
Jawel, kan wel, maar ik heb de unknownUser gemaakt voor ergens anders en dit is daar ook een use case voor vond ik

Reply:
Nvm I changed it